src/transformers/models/auto/modeling_auto.py

```diff
@@ -753,6 +753,32 @@
     ]
 )
 
+MODEL_FOR_IMAGE_TEXT_TO_TEXT_MAPPING_NAMES = OrderedDict(
```

Comment:
Should the existing classes be deleted from `MODEL_FOR_VISION_2_SEQ_MAPPING_NAMES` or do we keep them there?

Reply:
I just removed the ones that previously were in `IGNORE_NON_AUTO_CONFIGURED`.
Won't deleting them from  `MODEL_FOR_VISION_2_SEQ_MAPPING_NAMES` be a problem for BC?

Reply:
can also add llava-next-video and video-llava? Those two can work with image+text inputs, as well as video+text